pkgs/by-name/my/mycelium/package.nix

```diff
@@ -28,6 +29,12 @@ rustPlatform.buildRustPackage rec {
     darwin.apple_sdk.frameworks.SystemConfiguration
   ];
 
+  env = {
+    OPENSSL_NO_VENDOR = 1;
```

Comment:
I'm not too happy about openssl now being required, and opened https://github.com/threefoldtech/mycelium/issues/214 for clarification.

This however shouldn't block this PR.

Reply:
I added an item to my to-do list to check if we can easily replace that with rustls. But yeah I don't think we should block on this. 
Not sure if the current expression will build correctly on Darwin, I don't have much experience with these things on Darwin.

Reply:
I can kick off a darwin build in a sec.

Reply:
It successfully built on `aarch64-darwin` and `x86_64-darwin` :tada: